src/main/java/com/exasol/sql/dml/merge/rendering/MergeRenderer.java

```diff
@@ -1,6 +1,9 @@
 package com.exasol.sql.dml.merge.rendering;
 
-import com.exasol.sql.*;
```

Comment:
Three or more imports from the same package should be folded into `*`. I think the formatter is set up this way. Please check why this expansion happended. I am guessing an IntelliJ problem.